doc/Client-Portal Protocol.md

```diff
@@ -164,7 +166,8 @@ This a format for client reconnects.
           {
            id: string(StreamID),
            type: "forward" | "augmented" |"mixed",
-           media: object(MediaInfo),
+           media: object(MediaInfo) | null,
+           data: object(DataInfo) | null,
```

Comment:
Please add define of DataInfo

Reply:
Changed to boolean since `Publication.data` is boolean. Do you think it should be an object to describe the stream? I though a stream is always ordered and reliable. So it sounds like no more description is needed.

Reply:
Yes i agree it can be boolean only. How about move the data property to MediaInfo?

Reply:
Data stream could carry packets other than media frames, so I think it's not a part of media information.